examples/step-6/doc/intro.dox

```diff
@@ -167,6 +175,49 @@ module that explains how these constraints can be computed and what classes in
 deal.II work on them.
 
 
+<h3> How to deal with hanging nodes in practice </h3>
+
+The practice of hanging node constraints is rather simpler than the
+theory we have outlined above. In reality, you will really only have to
+add about half a dozen lines of additional code to a program like step-4
+to make it work with adaptive meshes that have hanging nodes. The
+interesting part about this is that it is entirely independent of the
+equation you are solving: How the constraints look like has nothing
+to do with the equation and only with the finite element in use.
+As a consequence, the code to deal with these constraints is entirely
+contained in the deal.II library itself, and you do not need to worry
+about the details.
+
+The steps you have to do to make this work are essentially like this:
+- You have to have a ConstraintMatrix object that stores the
+  constraints due our desire to keep the solution space continuous
+  even in the presence of hanging nodes.
+- You have to fill this object using the function
+  DoFTools::make_hanging_node_constraints().
```

Comment:
Similarly, I'd add the second half of the previous sentence here: "These constraints guarantee that the finite element solution will be continuous."